packages/graphql/src/graphql/ops/MSendPushNotifications.graphql

```diff
@@ -0,0 +1,3 @@
+mutation SendPushNotifications {
+  sendPushNotifications
```

Comment:
this will need to accept a secret as a parameter to ensure it can only be triggered from our own cronjon

Reply:
For documentation: we decided to pass the token as an HTTP header.